src/components/interfaces/MOBILE_API.xml

```diff
@@ -7121,6 +7212,70 @@
             </description>
         </param>
     </function>
+    <function name="GetInteriorVehicleDataConsent" functionID="GetInteriorVehicleDataConsentID" messagetype="request" since="6.0">
+        <param name="moduleType" type="ModuleType" mandatory="true">
+          <description>The module type that the app requests to control.</description>
+        </param>
+        <param name="moduleIds" type="String" maxlength="100" array="true" mandatory="true">
+          <description>Ids of a module of same type, published by System Capability. </description>
+       </param>
+    </function>
+    
+    <function name="GetInteriorVehicleDataConsent" functionID="GetInteriorVehicleDataConsentID" messagetype="response" since="6.0">
+        <param name="allowed" type="Boolean" array="true" mandatory="true">
```

Comment:
Referring to this comment in the rpc_spec PR https://github.com/smartdevicelink/rpc_spec/pull/187#discussion_r319178658, the `allowed` parameter should not be mandatory in the response
```suggestion
        <param name="allowed" type="Boolean" array="true" mandatory="false">
```

Reply:
@ShobhitAd Done in e238ec932f64fbc033e66c738128726cf021ee4c. Also has been added changes according to comments in the PR: https://github.com/smartdevicelink/rpc_spec/pull/187